docs/integrations/Trust-any-certificate.md

```diff
@@ -0,0 +1,101 @@
+# Trust Any Certificate
+
+## Overview
+
+Disabling SSL verification by setting `verify=False` in `BaseClient` (or by enabling “Trust Any Certificate” in an integration) allows connecting to servers using less-secure SSL configurations. This bypasses standard HTTPS checks—skipping certificate validation, lowering OpenSSL’s security level, and permitting older ciphers and renegotiation modes.
```

Comment:
two things here, setting `verify=False` in `BaseClient` is not interesting for customers who dont look at the code, this part should be removed.
We also are not **Disabling SSL verification**, SSL still happens, we are disabling **certificate validation** only, and using less "hard" ciphers, as well as some other things, such as allowing renegotiation

I think we shouldnt specify exactly what is lightened - we should say that we disable certificate checking and use less hardened ssl standards